gradle.properties

```diff
@@ -16,6 +16,8 @@ rsApi=3.0.0
 faker=1.0.2
 swagger=2.1.13
 jacksonVersion=2.13.1
+okHttp=4.9.3
```

Comment:
It is already at the top in this file. So can be removed.

Reply:
removed